### ISSUE TITLE
[WebToolsE2E][Aspire] On non-ENU OS, the description of the Aspire SDK component in the VS installer is not localized.

### ISSUE THREAD

**Issue by v-sherryfan** [dotnet/aspire#4558]
**REGRESSION INFO:** Reproduced on both VS 17.10 and 17.11

**INSTALL STEPS**
1. Clean machine: Win11 x64 22h2 JPN
2. Install latest released VS 17.10.2
    - Web workload

**ACTUAL**
Aspire SDK component descriptions are not localized
![image](https://github.com/dotnet/aspire/assets/71361488/abc3378b-6f92-4c65-8f1a-47d2a9f0ea31)

**NOTE:**
   - Reproduced on TRK / FRA / JPN OS.

**EXPECTED**
Aspire SDK component descriptions should be localized

**Comment by mitchdenny** [comment]
@joperezr PTAL.

**Comment by danmoseley** [comment]
@vijayrkn with Jose out, do you know where localized strings for this would come from? Perhaps it's not this repo.

Here, we only have this
https://github.com/dotnet/aspire/blob/20e641941f344b48061aec7dc5237c478e70e6d8/eng/workloads/workloads.csproj#L56

This may also be affected by workloads changes @joperezr  is working on.

**Comment by joperezr** [comment]
I don't think we currently localize this, even outside of the repo. @joeloff @timheuer do you know if this can get localized?

**Comment by radical** [comment]
Wasm does it like this:
https://github.com/dotnet/runtime/blob/5f52977c066a29d124694eb423eff0b866df37a7/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/localize/WorkloadManifest.pl.json#L1-L3 https://github.com/dotnet/runtime/tree/5f52977c066a29d124694eb423eff0b866df37a7/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/localize

**Comment by v-sherryfan** [comment]
@mitchdenny @joperezr This issue is not localized on all non-ENUs, and considering this is not friendly on localized VS 17.12. I see it was moved to the backlog, so can we increase the priority and solve it before VS 17.12 is released?

**Comment by maddymontaquila** [comment]
Template localization for non-ENU locales is now in place — `templatestrings.*.json` files exist for many locales under `src/Aspire.ProjectTemplates`. Closing as fixed.

**Comment by davidfowl** [comment]
Reopening after reviewing the recent bulk closure. The close comment appears to cite work that does not fully address this issue, so this should remain open for normal triage.
